### DISCUSSION TITLE
[Closed] Meteor Not working on Node version >14 .

### DISCUSSION THREAD

**Discussion by pacifiquem** [meteor/meteor#12547] [Feature Requests]
Anyone who knows why Meteor can't run on node version >14 ?

**Comment by StorytellerCZ** [comment]
Meteor is tied to a specific Node version ([see this blog post for more info on reasoning](https://blog.meteor.com/know-your-node-js-ff0b3a0f634e)). There is currently a massive effort ongoing right now to upgrade to Node 16 (#12359 #12015).
The main reason why Meteor 2.x can't just upgrade to Node 16+ is because Fibers, which enables everything to be sync code in Meteor, is not working in Node 16+.

Meteor 2.x is tied to Node 14 and is currently being prepared for easy migration to Meteor 3.0 which will be tied to Node 16+

**Reply by pacifiquem** [reply]
Okay that's Great .

**Comment by fredmaiaarantes** [comment]
Meteor 3.0 is here with Node.js 20! 🎉
https://dev.to/meteor/meteor-3-is-officially-here-3gml

**Comment by pacifiquem** [comment]
Good job @fredmaiaarantes
